.env.sample

```diff
@@ -1,5 +0,0 @@
-REACT_APP_CLIENT_ID=example
```

Comment:
yeah this is the example that needs to be in the github, there isn't anything in there that is specific to our project. So just add that back and then push again and you should be good to go.